app/app.go

```diff
@@ -181,9 +188,10 @@ var _ servertypes.Application = (*IxoApp)(nil)
 
 // Extended ABCI application
 type IxoApp struct {
-	*baseapp.BaseApp  `json:"_bam_base_app,omitempty"`
-	legacyAmino       *codec.LegacyAmino      `json:"legacy_amino,omitempty"`
-	appCodec          codec.Marshaler         `json:"app_codec,omitempty"`
+	*baseapp.BaseApp `json:"_bam_base_app,omitempty"`
+	legacyAmino      *codec.LegacyAmino `json:"legacy_amino,omitempty"`
+	// TODO: Confirm is this is correct
+	appCodec          codec.Codec             `json:"app_codec,omitempty"`
```

Comment:
@ig-shaun @demondayza @steveixo 
Cosmos Restructured this Json Codec, but this also seems this solution also seems to be deprecated in the future.
For now this works as intended.